test/contrib/nexus/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+set(SOURCES attribute_h5py_bool_test.cpp
+            attribute_pniio_bool_test.cpp
+            dataset_h5py_bool_test.cpp
+            dataset_pniio_bool_test.cpp
+            ebool_test.cpp)
+
+add_executable(ebool_test ${SOURCES})
+target_link_libraries(ebool_test h5cpp $<$<AND:$<CXX_COMPILER_ID:GNU>,$<VERSION_LESS:$<CXX_COMPILER_VERSION>,11.0>>:stdc++fs>)
```

Comment:
Generator expressions break the pkg-config support. It would good to change them to standard if statements.

Reply:
> Generator expressions break the pkg-config support. It would good to change them to standard if statements.

Ahh, It is in tests so it does not matter. So here everything is OK.